src/test/kotlin/tornadofx/testapps/SwitchViewApp.kt

```diff
@@ -11,11 +11,13 @@ class GotoContainerView : View("ContainerView") {
 
     override val root = borderpane {
         top {
-            button("Goto view").setOnAction {
-                if (center.lookup("#view1") != null)
-                    gotoSubView1.goto(gotoSubView2)
-                else
-                    gotoSubView2.goto(gotoSubView1)
+            button("Goto view") {
```

Comment:
Sweet, I merged and fixed those and incorporated even more small syntax improvements.